storybookjs/storybook
# fix(clipboard): try to fix clipboard issue

Issue: #16642

## What I did

I just added the property allow clipboard-write (according to the doc) on the iframe responsible to display the Source Code, so the user can copy.

## How to test

I'm not sure how to test it. We need two storybook with composition using chromatic to see if it works or not. However, this small fix should not destroy all Storybook :)


If your answer is yes to any of these, please make sure to include it in your PR.
